ecommerce/settings/production.py

```diff
@@ -84,3 +84,5 @@ def get_env_setting(setting):
 # END PAYMENT PROCESSOR OVERRIDES
 
 ENTERPRISE_API_URL = urljoin(ENTERPRISE_SERVICE_URL, 'api/v1/')
+
+OPENEDX_EXTENSIONS_API_URL = environ.get('OPENEDX_EXTENSIONS_API_URL', '/camrom/api/v1/')
```

Comment:
is this the proper way to do it? @jfavellar90 I'd still need to add this setting the customer data.